<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WinnerDialog</class>
 <widget class="QDialog" name="WinnerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="winner_dialog">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Player 0 has won the game! </string>
   </property>
  </widget>
  <widget class="QLabel" name="p1_wins_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>PLayer 1 has 0 win(s)</string>
   </property>
  </widget>
  <widget class="QLabel" name="p2_wins_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>PLayer 2 has 0 win(s)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="playAgainButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Rematch!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Rage Quit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
